formatTest/typeCheckedTests/expected_output/arityConversion.re

```diff
@@ -16,11 +16,11 @@ module Test = {
     | Or (int, int);
 };
 
-Test.And (1, 2);
+let _ = Test.And (1, 2);
```

Comment:
The 4.02 printer hasn't changed, so the roundtrip still loses information. However later versions should work as expected.